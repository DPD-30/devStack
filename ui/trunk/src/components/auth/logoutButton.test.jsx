import React from 'react'
import { renderToString } from 'react-dom/server'


import ComponentAuthLogoutButton from './logoutButton.jsx'

jest.unmock('./logoutButton.jsx')


test('ComponentAuthLogoutButton matches snapshot', () => {
	const comp = renderToString(<ComponentAuthLogoutButton />)

	expect(comp).toMatchSnapshot()
})
