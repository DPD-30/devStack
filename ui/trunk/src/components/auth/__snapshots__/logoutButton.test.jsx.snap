// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentAuthLogoutButton matches snapshot 1`] = `
<button class="btn btn-secondary"
        data-reactroot
>
  SIGN OUT
</button>
`;
